barretenberg/cpp/scripts/ultra_honk_tests.sh

```diff
@@ -0,0 +1,22 @@
+set -eu
```

Comment:
Simple script that's useful for running tests locally. Will delete in a future PR.